src/lib/components/pricing/faqs.svelte

```diff
@@ -1,141 +1,36 @@
 <script lang="ts">
-  import { isEurope } from "$lib/utils/helpers";
   import Section from "../section.svelte";
   import Faqs from "../faqs/faqs.svelte";
   import Faq from "../faqs/faq.svelte";
   import Modal from "../modal.svelte";
   import RedeemOffer from "./redeem-offer.svelte";
 
+  import { pricingFAQ } from "$lib/contents/pricing";
+
   let isModalOpen: boolean = false;
 </script>
 
 <style lang="scss">
   h2 {
     text-align: center;
   }
-
-  .images {
+  :global(.images) {
     display: flex;
     margin-top: var(--xx-small);
-
-    & > *:not(:last-child) {
-      margin-right: var(--xx-small);
-    }
+  }
+  :global(.images > *:not(:last-child)) {
```

Comment:
This resolves the issue with the aligned images. Is it clever to use the `:global` here though 🤔

Reply:
@Cahllagerfeld You can scope the global to this component via wrapping the faqs in a div and then use: 
```suggestion
  div :global(.images > *:not(:last-child)) {
```
This way it won't have undesired impacts on things.